core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -54,6 +59,30 @@ object Boot extends App with LazyLogging {
       slickDataSource.initWithLiquibase(liquibaseChangeLog, changelogParams)
     }
 
+    val metricsConf = conf.getConfig("metrics")
+    val metricsPrefix = {
+      val basePrefix = metricsConf.getString("prefix")
+      if (metricsConf.hasPath("includeHostname") && metricsConf.getBoolean("includeHostname")) {
```

Comment:
`EnhancedScalaConfig` defines some helpers that might be nicer here, like `getBooleanOption`